app/assets/javascripts/application.js

```diff
@@ -1,3 +1,13 @@
+//= require govuk_publishing_components/dependencies
 //= require govuk_publishing_components/lib
+//= require govuk_publishing_components/components/button
+//= require govuk_publishing_components/components/cookie-banner
+//= require govuk_publishing_components/components/cross-service-header
 //= require govuk_publishing_components/components/error-summary
 //= require govuk_publishing_components/components/radio
+//= require govuk_publishing_components/components/feedback
+//= require govuk_publishing_components/components/layout-super-navigation-header
+//= require govuk_publishing_components/components/search-with-autocomplete
+//= require govuk_publishing_components/components/skip-link
+
+//= require govuk_web_banners/dependencies
```

Comment:
The last commit says `Add JS used by global banners` but doesn't include any JS changes, are those scripts already included with this line?

Reply:
Done!